### ISSUE TITLE
ServiceReferenceWrapper

### ISSUE THREAD

**Issue by koderi-dp** [loekd/ServiceFabric.PubSubActors#41]
Hi,

Found some issue with IServiceProxyFactory when i did some tests and debugging on local fabrics, dev 1 node. if  you example kill broker service, it will load all references from state in my case ServiceReferenceWrapper, but IServiceProxyFactory will be null, and when queue is executing PublishAsync you will get null reference, so for now this will resolve it, and in publish async use ServiceProxy

```
 [IgnoreDataMember]
	    public IServiceProxyFactory ServiceProxy
        {
            get => serviceProxyFactory ?? (serviceProxyFactory = new ServiceProxyFactory());
        }
```

**Comment by loekd** [comment]
Thank you for reporting this! 
The issue is fixed in 7.3.5.
Please let me know if it works for you.

**Comment by koderi-dp** [comment]
Working fine
